soundfile.py

```diff
@@ -968,6 +990,120 @@ def read(self, frames=-1, dtype='float64', always_2d=False,
                 out[frames:] = fill_value
         return out
 
+    def read_raw(self, frames=-1, dtype=None):
```

Comment:
I think that the `dtype` parameter is confusing here. Maybe instead of having both `frames` and `dtype` a simple `numbytes` would be preferable.